crt_portal/cts_forms/tests/test_forms.py

```diff
@@ -745,19 +746,51 @@ def setUp(self):
             report.save()
 
         # generate reports with no email address
-        reports_email_none = [Report.objects.create(**SAMPLE_REPORT) for _ in range(8)]
+        reports_email_none = [Report.objects.create(**SAMPLE_REPORT) for _ in range(7)]
```

Comment:
This is starting to feel cumbersome, list comprehensions to generate sets of Reports and then iterating through them again to modify their values, flagging for us to address in another issue.

Reply:
https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/846